GentApp/ViewModels/BranchViewModel.cs

```diff
@@ -16,19 +16,33 @@ namespace GentApp.ViewModels
 	public class BranchViewModel : ViewModelBase
 	{
 		private readonly ILogger log = LogManagerFactory.DefaultLogManager.GetLogger<BranchViewModel>();
-		private readonly CompanyService companyService = new CompanyService();
-		private readonly PromotionService promotionService = new PromotionService();
-		private readonly BranchService branchService = new BranchService();
-		private readonly EventService eventService = new EventService();
 		private readonly INavigationService _navigationService;
-		
-		public BranchViewModel(Helpers.INavigationService navigationService)
+
+		public BranchViewModel(INavigationService navigationService)
 		{
 			_navigationService = navigationService;
 		}
 
-		private IEnumerable<Promotion> _promotions;
-		public IEnumerable<Promotion> Promotions
+		public CompanyViewModel CompanyViewModel {
+			get {
+				return SimpleIoc.Default.GetInstance<CompanyViewModel>();
+			}
+		}
+
+		private RelayCommand<string> _saveBranchCommand;
+
+		public RelayCommand<string> SaveBranchCommand {
+			get {
+				return _saveBranchCommand = new RelayCommand<string>(name => {
+					CompanyViewModel.EditBranch(Events, Promotions, name);
+					RaisePropertyChanged(nameof(Events));
+					RaisePropertyChanged(nameof(Promotions));
```

Comment:
.

Reply:
Ik denk nie da branchviewmodel een selectedbranch heeft. En editbranch callt die al denk ik in companyviewmodel

Reply:
ah ok

Reply:
Ah nee hij doet da nie